### PR TITLE
Rename this gem to Buildkite::TestCollector

### PR DESCRIPTION
**Pull request by JuanitoFatas**
We decided on `buildkite-test_collector` and it follows the [Rubygems naming convention](https://guides.rubygems.org/name-your-gem/).
 
[View Rendered README.md](https://github.com/buildkite/test-collector-ruby/blob/rename/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,100 @@
-# RSpec Buildkite Analytics
+# Buildkite Collectors for Ruby
 
-This gem collects data about your test suite's performance and reliability, and allows you to see trends and insights about your test suite over time ✨
+Official [Buildkite Test Analytics](https://buildkite.com/test-analytics) collectors for Ruby test frameworks ✨
 
-## Installation
+⚒ **Supported test frameworks:** Jest, and [more coming soon](https://github.com/buildkite/test-collector-ruby/issues?q=is%3Aissue+is%3Aopen+label%3A%22test+frameworks%22).
```
Comment (swebb) [inline README.md:5]:
```suggestion
⚒ **Supported test frameworks:** RSpec and Minitest.
```

File: README.md [hunk 2]
```diff
@@ -1,51 +1,100 @@
...
-```sh
-git checkout -b install-buildkite-test-analytics
-```
+## 👉 Installing
+
+### Step 1
+
+[Create a test suite](https://buildkite.com/docs/test-analytics), and copy the API token that it gives you.
+
+Add the [`buildkite-test_collector` gem](https://rubygems.org/gems/buildkite-test_collector):
```
Comment (swebb) [inline README.md:15]:
```suggestion
Add the [`buildkite-test_collector`](https://rubygems.org/gems/buildkite-test_collector) gem:
```

File: README.md (hunk 1 above)
Comment (JuanitoFatas) [reply README.md]: [94ea11a](https://github.com/buildkite/test-collector-ruby/pull/107/commits/94ea11a153f35f3e6e69a2130991e0225aa6d42b)

File: README.md (hunk 2 above)
Comment (JuanitoFatas) [reply README.md]: [93b5da0](https://github.com/buildkite/test-collector-ruby/pull/107/commits/93b5da0b42ab8ad7cc5b24da6d0aa44e9c3af8e8)

**Review by gchan — APPROVED** [review]
📃 🌟

**Pull request merged**